juicy-ace-editor.html

```diff
@@ -87,7 +87,12 @@
             } else {
                 // container.appendChild(text);
                 container.textContent = this.value;
-                editor = ace.edit(container);
+                const options = {};
+                // support autoresizing
+                this.hasAttribute('max-lines') && (options.maxLines = parseInt(this.getAttribute('max-lines'), 10));
+                this.hasAttribute('min-lines') && (options.minLines = parseInt(this.getAttribute('min-lines'), 10));
```

Comment:
Maybe `Number` is better than `parseInt`. 

https://www.reddit.com/r/learnjavascript/comments/5x81fy/numberparseint_vs_parseint/degea65/